src/backend/core/api/viewsets.py

```diff
@@ -886,3 +889,20 @@ def perform_create(self, serializer):
         invitation.document.email_invitation(
             language, invitation.email, invitation.role, self.request.user
         )
+
+class ConfigView(views.APIView):
+    """API ViewSet for sharing some public settings."""
+
+    permission_classes = [AllowAny]
+
+    def get(self, request):
+        """
+        GET /api/v1.0/config/
+            Return a dictionary of public settings.
+        """
+        array_settings = ["SENTRY_DSN", "DJANGO_CONFIGURATION"]
+        dict_settings = {}
+        for setting in array_settings:
+            dict_settings[setting] = os.environ.get(setting, None)
```

Comment:
I think it's better not to shortcircuit Django Configuration settings so that we are forced to declare our settings properly and document them. This would be a bit too wild IMO.
Also we want to keep a common pattern with our other projects e.g https://github.com/numerique-gouv/people/blob/main/src/backend/core/api/viewsets.py#L501

Reply:
I separated the backend in this PR: https://github.com/numerique-gouv/impress/pull/425